selfdrive/test/process_replay/process_replay.py

```diff
@@ -25,25 +27,27 @@
 TIMEOUT = 15
 PROC_REPLAY_DIR = os.path.dirname(os.path.abspath(__file__))
 FAKEDATA = os.path.join(PROC_REPLAY_DIR, "fakedata/")
-
+VIPC_STREAM = {"roadCameraState": VisionStreamType.VISION_STREAM_ROAD, "driverCameraState": VisionStreamType.VISION_STREAM_DRIVER,
```

Comment:
this dict is in a bunch of places, we should move it somewhere common

Reply:
I've added helpers for handling vision related metadata (camera states, streams, dt, frame sizes) to `vision_meta.py`.